jbpm-test-coverage/src/test/java/org/jbpm/test/functional/timer/GlobalQuartzRAMTimerServiceTest.java

```diff
@@ -59,7 +61,7 @@ public void setUp() {
     }
     
     @After
-    public void tearDown(){
+    public void tearDown() throws IOException, SQLException {
```

Comment:
Because I've already included a call to the reusable class in charge of creating the DB schemas: ScriptsBase.executeScriptRunner(DB_DDL_SCRIPTS_RESOURCE_PATH, false); from the parent class TimerBaseTest

Though, I think for the time being I will comment it out as this piece of work is still in progress (works fine, but some tests are still failing due to many different reasons) and will be delivered in next PRs.

What do you think? I mean, there is no harm of having this call here but I'm open to comment it out for the moment.

Reply:
I see... Though now when you look at it, we call a method (tearDownOnce()) which has @BeforeClass annotation on it also in @Before method. I think it would be good to make it more clear. But that is just cosmetics.

However, I see that now the schema is created/scripts are run in `GlobalQuartzRAMTimerServiceTest` which stores timers in RAM rather than DB, but there are no such operations in `GlobalQuartzDBTimerServiceTest`. Shouldn't it be the other way round that we should create schema in DB tests and for RAM it is not needed?

Reply:
I would say tearDownOnce() call in @before method is not needed there, since it will be called in the @After method. I think the tearDownOnce() call is needed in the @After method because it is supposed to be a RAM based database (i.e. database is created and destroy after every test method).

But yes, it shouldn't create/run any SQL script for this case.

In the other hand, GlobalQuartzDBTimerServiceTest, the SQL scripts are created/destroyed once in the @AfterClass and @BeforeClass respectively in the class TimerBaseTest.

However all of these changes are still in progress and will be delivered in the next upcoming PR.

What do you think?

Reply:
Understand. Then I guess it would be probably good to not alter these classes at all? So leave them as they were, do only necessary changes so we don't mix it?

Reply:
Agree.

Will revert unnecessary changes for now.

Reply:
Done.